src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -54,26 +54,32 @@ public Utf8JsonReader(in ReadOnlySequence<byte> jsonData, bool isFinalBlock, Jso
             if (jsonData.IsSingleSegment)
             {
                 _nextPosition = default;
-                _currentPosition = default;
+                _currentPosition = jsonData.Start;
                 _isLastSegment = isFinalBlock;
                 _isSingleSegment = true;
             }
             else
             {
-                _nextPosition = jsonData.Start;
+                _currentPosition = jsonData.Start;
```

Comment:
This branch of the code might warrant some comments. There's currentPosition, nextPosition and previousNextPosition 😄

Reply:
This code/pattern is similar to the one in `SequenceReader`:
https://github.com/dotnet/corefx/blob/ed0a57d42d859a4b8fee19441c504bc296c2a2ae/src/System.Memory/src/System/Buffers/SequenceReader.cs#L212-L235

I see value in adding comments to clarify (and will add them in future PRs), but I don't want to reset CI just for this.

Reply:
> I don't want to reset CI just for this.

If you're just adding comments and nothing else, you could just add a commit with the comments, double and triple check that all added lines begin with //, and you don't need to wait for CI to finish. :)